app/views/users/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Please Insert Your Log-in information</h2>
+
+<%# <h3>form with html</h3>
+<form action="/users" method="POST" accept-charset="UTF-8">
+       <label for="username">Username:</label>
+       <input type="text" id="username" name="user[username]"><br>
+       <label for="email">Email:</label>
+       <input type="text" id="email" name="user[username]"><br>
+       <label for="password">Password:</label>
+       <input type="text" id="password" name="user[username]"><br>
+      <%#  <<input type="submit" value="Submit">
+</form> %>
```

Comment:
Your commented out form was expected to make use of the `form_authenticity_token`. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form2.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms)) This was aimed at helping you appreciate its relevance to forms.